geomstats/special_orthogonal_group.py

```diff
@@ -672,23 +678,31 @@ def matrix_from_tait_bryan_angles(self, tait_bryan_angles,
                              ' for rotations in %d dimensions.' % self.n)
         tait_bryan_angles = gs.to_ndarray(tait_bryan_angles, to_ndim=2)
 
-        extrinsic_zyx_or_intrinsic_xyz = (
-                       (extrinsic_or_intrinsic == 'extrinsic'
-                        and order == 'zyx')
-                       or (extrinsic_or_intrinsic == 'intrinsic'
-                           and order == 'xyz'))
-
-        extrinsic_xyz_or_intrinsic_zyx = (
-                       (extrinsic_or_intrinsic == 'extrinsic'
-                        and order == 'xyz')
-                       or (extrinsic_or_intrinsic == 'intrinsic'
-                           and order == 'zyx'))
-        if extrinsic_zyx_or_intrinsic_xyz:
+        extrinsic_zyx = (extrinsic_or_intrinsic == 'extrinsic'
```

Comment:
could you do a check like assert order in ('zyx', 'xxx')
extrinsic or intrinsic in ('foo', 'bar')